app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -126,6 +128,20 @@ class MembersFragment : BaseFragment() {
                                 adapter = rvAdapter
                                 isRecyclerView = true
                             }
+                        } else {
+                            if (!filterMap["location"].isNullOrEmpty()) {
+
+                                // Here we are checking if location from filter
+                                    // matches with location from any user's profile
+                                val check_location = membersViewModel.userList.any {
```

Comment:
can this be in camelCase just for consistency with the rest of the code :)
little suggestion: `hasUsersWithLocation` but it's fine if it's `checkLocation` as well

Reply:
Why do we need it as a separate variable though? it's only being used in one place. We could easily inline it. And it's not like it's a constant anyways.

Reply:
I suggested that not because it made the code work any better, just for a matter of readability. I had a hard time reading what was going on, I could imagine someone else having it as well. It was a lot of logic inside an if statement. @epicadk

Reply:
Yep that makes sense. It was just a nit. Progaurd would handle that anyways.

Reply:
yap totally nit 🤓